mods/resize-text.user.js

```diff
@@ -0,0 +1,487 @@
+        const userNotifsElemH1 = userNotifsSelect.querySelectorAll('h1, label[for="message_body"]');
+
+        userNotifsElem.forEach(userNotifResize => {
+            userNotifResize.style.setProperty('font-size', fontSizes.userNotifs);
+        })
+
+        userNotifsElemH1.forEach(userNotifResizeH1 => {
+            userNotifResizeH1.style.setProperty('font-size', `${notifSizeMultiply}px`);
+        })
+
+        notifButtons.forEach(notifButtonResize => {
+            const notifPurge = notifButtonResize.querySelectorAll('.btn.btn__secondary span');
+
+            notifButtonResize.style.setProperty('font-size', fontSizes.userNotifs);
+
+            notifPurge.forEach(notifPurgeResize => {
+                notifPurgeResize.style.setProperty('font-size', fontSizes.userNotifs);
+            })
+        })
+
+    })
+
+// === SORT BY === // 
+
+    // sort by *variables*
+    const sortBy = document.querySelectorAll('aside#options menu li a, aside#options menu i');
+
+    // sort by *loops*
+    sortBy.forEach(sortByElem => {
+        sortByElem.style.setProperty('font-size', fontSizes.sortBy);
+    })
+
+// === Footer === // 
+
+    //footer *variables*
+    const footerUseful = document.querySelectorAll('footer#footer');
+    const footerMultiply = parseFloat(settings["optionFooter"]) * 1.5;
+
+    //footer *loops*
+    footerUseful.forEach(footerSelect => {
+        const footerElems = footerSelect.querySelectorAll('section menu li a, section div a, div li a, i, select[data-action="kbin#changeLang"], #text');
+        const footerH1 = footerSelect.querySelectorAll('section h5');
+
+        footerElems.forEach(footerResize => {
+            footerResize.style.setProperty('font-size', fontSizes.footer);
+        });
+
+        footerH1.forEach(footerH1Resize => {
+            footerH1Resize.style.setProperty('font-size', `${footerMultiply}px`);
+        })
+    })
+
+// === ACTIVITY === //
+    const activity = document.querySelectorAll('div.section.users.users-columns');
+
+    // create posts *loops*
+    activity.forEach(activityElem => {
+        const activityElement = activityElem.querySelectorAll('ul, li, small, a, img');
+
+        activityElement.forEach(activityResize => {
+                activityResize.style.setProperty('font-size', fontSizes.activity);
+        })
+    });
+
+// === OPACITY FUNCTIONALITY === //
+    let opacity = settings.opacity;
+    const kesModalContent = document.querySelector('div.kes-settings-modal-content');
+    const kesModalContainer = document.querySelector('div.kes-settings-modal-container');
+
+    kesModalContent.style.setProperty('background-color', `rgba(44, 44, 44, ${opacity / 10})`);
+    kesModalContainer.style.setProperty('background-color', `rgba(44, 44, 44, ${opacity / 10})`);
+
+
+} // end of resizeText() function //
+
+
+function textResize(toggle) {
+    if (toggle) {
+        resizeText();
+    } else {
+        document.removeEventListener('click', eventListenerCheckbox);
```

Comment:
I couldn't find eventListenerCheckbox being used anywhere, either. Is this old code? On the subject of switching off the script without reloading: I got it working on my local copy, it is fairly easy:

Think about what we do when the "reset" button is clicked: it flushes all of the custom values and sets the settings to the `initial` value, then enters the `textResize` function with a value of `true` and steps through every element and sets it to the new size, i.e., the initial size. So what do you think we need to do when it's turned off? Same thing. Rather than catch a reset event, toggling the button off just sends `false` to the `textResize` function. So what should that branch do? Just step through the `resizeText()` function again, but this time set everything to the initial values. Here's how it might end up looking (let's just pretend that the initial size is 14px for everything):

```
function setResizeText (param) {
    // === FONT SIZE SETTINGS OBJ === //
    if (param === "custom") {
        const settings = getModSettings('resize');
        const fontSizes = {
            header: `${settings["optionHeader"]}px`,
            posts: `${settings["optionPosts"]}px`,
            magSidebar: `${settings["optionMagSidebar"]}px`,
            homeSidebar: `${settings["optionHomeSidebar"]}px`,
            profile: `${settings["optionProfile"]}px`,
            createPosts: `${settings["optionCreate"]}px`,
            comments: `${settings["optionComments"]}px`,
            userSettings: `${settings["optionUserSettings"]}px`,
            userMessages: `${settings["optionMessages"]}px`,
            userNotifs: `${settings["optionNotifs"]}px`,
            sortBy: `${settings["optionSortBy"]}px`,
            footer: `${settings["optionFooter"]}px`,
            activity: `${settings["optionActivity"]}px`
        };
        walkThroughFonts(fontSizes);
    } else {
        const fontSizes = {
            header: `14px`,
            posts: `14px`,
            magSidebar: `14px`,
            homeSidebar: `14px`,
            profile: `14px`,
            createPosts: `14px`,
            comments: `14px`,
            userSettings: `14px`,
            userMessages: `14px`,
            userNotifs: `14px`,
            sortBy: `14px`,
            footer: `14px`,
            activity: `14px`
        }
        walkThroughFonts(fontSizes);
    }
}
function textResize (toggle) {
    if (toggle) {
        setResizeText("custom");
    } else {
        setResizeText("default");
    }
}
```

I don't think you need to change this whole part yourself, I'll just commit my edits into your fork and we can test it again.